Makefile

```diff
@@ -17,6 +17,7 @@ all: release
 release:
 	docker build --no-cache -t aws-fluent-bit-plugins:latest -f Dockerfile.plugins .
 	docker build -t amazon/aws-for-fluent-bit:latest -f Dockerfile .
+	docker build -t amazon/aws-for-fluent-bit-init:latest -f Dockerfile.init .
```

Comment:
I thought/think the image should be `amazon/aws-for-fluent-bit:latest-init` or `amazon/aws-for-fluent-bit:init-latest`. 

Building the image with this tag: `amazon/aws-for-fluent-bit-init` implies that we have a Docker Hub repo with that name. But we do not. I beleive we are planning to push these images to the same repos as we use for the non-init image right? And thus only the tag is different. So the same hsould be true in our build script IMO.

Reply:
So unfortunately creating a new Docker Hub repo is a whole big process... also while I understand the tag becomes very long, I also think its better if all AWS for Fluent Bit images are in one repo if possible. 

A good example of long tags is fluentd repo which has tons of tags: https://hub.docker.com/r/fluent/fluentd/tags

Reply:
Yea I already discussed this with @zhonghui12 .
Understand to create a new Docker Hub repo will be a big process..

We finally decided to use same repo and these tags:
`amazon/aws-for-fluent-bit:init-latest`
` amazon/aws-for-fluent-bit:init-2.26.0`
`amazon/aws-for-fluent-bit:init-amd64-2.26.0`